uni/lib/view/exams/exams.dart

```diff
@@ -91,20 +91,25 @@ class ExamsPageViewState extends GeneralPageViewState<ExamsPageView> {
       child: createExamsCards(context, exams),
     );
   }
-
   Widget createExamsCards(context, List<Exam> exams) {
     final List<Widget> examCards = <Widget>[];
     examCards.add(DayTitle(
         day: exams[0].begin.day.toString(),
         weekDay: exams[0].weekDay,
         month: exams[0].month));
     for (int i = 0; i < exams.length; i++) {
-      examCards.add(createExamContext(context, exams[i]));
+      examCards.add(ExamContext(exams[i]));
```

Comment:
As per the meeting, consider rewriting this in a more declarative way (e.g. `exams.map(...)`).

Reply:
When changed to a more declarative way, it creates an type conflict with the DayTitle element.

Reply:
Well, something like return [DayTitle, ...map().toList()] should work, since they are all Widgets. Am I missing something?

Reply:
It still does not work. When the toList() method is called turns the elements into a List<Widget> and without it the elements are Iterable<Widget>, for it to work it would be needed to convert back to Widget which makes it slower and redundant.